common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/MicrosoftAuthorizationRequest.java

```diff
@@ -58,12 +59,14 @@
     /**
      * Can be used to pre-fill the username/email address field of the sign-in page for the user, if you know their username ahead of time.
      */
+    @Expose()
```

Comment:
PII?

Reply:
Wasn't sure about this myself.  Am expecting a document that may shed some additional light.

Reply:
upn and login hint are pii.